src/utilities/util.ts

```diff
@@ -580,6 +580,59 @@ export async function requestInitOperatorCollectionInfo(): Promise<string[] | un
   return args;
 }
 
+/**
+ * Prompts the user for the necessary info to create a Credential Secret
+ * @returns - A Promise containing the list of parameters to pass to the command
+ */
+export async function requestCreateCredentialSecreteInfo(): Promise<string[] | undefined> {
+  const args: Array<string> = [];
+  const singleInputRegex = /^\S*$/; // matches single input, no spaces
+
+  const validateSingleInput = (text: string): string | null => {
+    const isValid = text.trim().match(singleInputRegex)?.length;
+    return isValid ? null : "Please enter a single input for this field.";
+  };
+
+  const operatorName = await vscode.window.showInputBox({
```

Comment:
Can we just retrieve this automatically from the `operator-config`?

Reply:
Yea, good thinking. I took it a step further and checked for installed operators in the namespace because the pod is necessary anyways. Let me know if it you think just pulling the name is a better alternative.